deploy/crds/argoproj.io_argocds_crd.yaml

```diff
@@ -10,998 +10,1016 @@ spec:
     plural: argocds
     singular: argocd
   scope: Namespaced
-  subresources:
```

Comment:
Can you please remove formatting changes and only add changes relevant to the new features?   It is impossible to review and has a higher risk to introduce bugs.

Reply:
changed the required and re-ran operator-sdk generate crds